pvl_hsu_soiling.m

```diff
@@ -0,0 +1,351 @@
+function SR = pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10, varargin)
+% PVL_HSU_SOILING Calculates soiling rate over time given particulate and rain data
+%
+% Syntax
+%   [SR]=pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10)
+%   [SR]=pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10, ModelType)
+%   [SR]=pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10, ModelType, RainAccPeriod)
+%   [SR]=pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10, ModelType, RainAccPeriod, LUC, WindSpeed, Temperature)
+%
+% Description  
+%   [SR]=pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10)
+%      Uses the time, rainfall amounts, tilt angle, particulate matter
+%      concentrations, and rain cleaning threshold to predict soiling rate
+%      using the fixed settling model. Uses hourly rain accumulation to
+%      determine cleaning.
+%   [SR]=pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10, ModelType)
+%      Uses the time, rainfall amounts, tilt angle, particulate matter
+%      concentrations, and rain cleaning threshold to predict soiling rate
+%      using the model type selected by ModelType. Uses hourly rain
+%      accumulation to determine cleaning.
+%   [SR]=pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10, ModelType, RainAccPeriod)
+%      Uses the time, rainfall amounts, tilt angle, particulate matter
+%      concentrations, and rain cleaning threshold to predict soiling rate
+%      using the model type selected by ModelType. The user specifies the
+%      rain accumulation period (in hours) over which to determine whether
+%      sufficient rain has fallen to clean the module.
+%   [SR]=pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10, ModelType, RainAccPeriod, LUC, WindSpeed, Temperature)
+%      Uses the parameters of the fixed models along with Land Use Category
+%      (LUC), wind speed, and ambient temperature to predict soiling ratio
+%      using the variable deposition velocity model.
+%
+% Input Parameters:
+%   Time is a struct with elements as described in pvl_maketimestruct. 
+%     Time values for the soiling function do not need to be 
+%     regularly spaced, although large gaps in timing are discouraged.
+%
+%   Rain is a vector of rainfall values of the same length as the fields of
```

Comment:
`Rain is a vector of rainfall in mm of the same length as Time

Reply:
Corrected.